11_webui.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+
+set -eux
+source common.sh
+
+export UIPATH="$GOPATH/src/github.com/kubevirt/web-ui-operator"
+
+oc new-project kubevirt-web-ui
+cd $UIPATH/web-ui-operator/deploy
+oc apply -f service_account.yaml
+oc adm policy add-scc-to-user anyuid -z kubevirt-web-ui-operator
+oc apply -f role.yaml
+oc apply -f role_binding.yaml
+oc apply -f role_binding_kube-public.yaml
+oc apply -f crds/kubevirt_v1alpha1_kwebui_crd.yaml
+oc apply -f operator.yaml
+echo -e "  openshift_master_default_subdomain: apps.ostest.test.metalkube.org\n  public_master_hostname: api.ostest.test.metalkube.org:6443" >> deploy/crds/kubevirt_v1alpha1_kwebui_cr.yaml
```

Comment:
Maybe instead of hardcoding domain's part better to reference it via *CLUSTER_DOMAIN* variable defined in [ocp_install_env.sh](#https://github.com/openshift-metalkube/dev-scripts/blob/master/ocp_install_env.sh#L6), 

```suggestion
echo -e "  openshift_master_default_subdomain: apps.${CLUSTER_DOMAIN}\n  public_master_hostname: api.${CLUSTER_DOMAIN}:6443" >> deploy/crds/kubevirt_v1alpha1_kwebui_cr.yaml
```

wdyt?

Reply:
Yeah we should use the variable here.

I also wonder if we should perhaps combine this with the 09_deploy_manifests.sh script that deploys the other kubevirt stuff, maybe we rename that and do all kubevirt things there?